apps/condo/domains/ticket/schema/CreateResidentTicketService.test.js

```diff
@@ -110,25 +110,6 @@ describe('CreateResidentTicketService', () => {
         expect(contact.id).toMatch(UUID_RE)
     })
 
-    test('resident: do not create contact when create resident ticket with contact with same data', async () => {
```

Comment:
why you want to remove this test?

Reply:
`CreateResidentTicket` is a deprecated service, I removed it in another pr